src/components/structural/header/ProjectView.js

```diff
@@ -351,6 +356,13 @@ class Project extends React.Component {
                 </ListItemIcon>
                 <ListItemText inset primary="Send" />
             </MenuItem>
+            <MenuItem
+                onClick={() => this.props.exportFunc(this.props.user.uid, this.state.projectId)}>
+                <ListItemIcon>
+                    <Icon className="material-icons">download</Icon>
+                </ListItemIcon>
+                <ListItemText inset primary="Export Scene" />
+            </MenuItem>
```

Comment:
Example project also has an option to export the scene but unable to.

Reply:
Exporting example scene still gave 404 error. Is this intentional or do we want to give user option to export the example scene?